README.adoc

```diff
@@ -1,50 +1,31 @@
-# Project Starfish
+# Project Starfish 
 
-GSOC 2020 Starfish POC
+A  cross platform client side application that would let you open a file in IDE of your choice (vscode, eclipse, intellij, emacs, vi, etc.) or start a debugger locally directly from the browser or file system.This will enable an option to “Open in IDE” through browser extension on various websites like github, gitlab,gitbucket etc similar to one you see in this demo below.
```

Comment:
I know I used this as an example before but "or start a debugger" is very specific and we might not do that as the first thing - I suggest to make it a bit more open ended, like "and other commands locally".